### PR TITLE
Faro for AnVIL Prod

### PR DESCRIPTION
**Pull request by k-burt-uch**
Link to Jira ticket if there is one:

### Environments
- AnVIL Prod

### Description of changes
- Add Faro

### CODE REVIEW

**Comment by jawadqur** [comment]
We also need to bump portal to `5.33.1`

**Comment by k-burt-uch** [comment]
> We also need to bump portal to `5.33.1`

Merged https://github.com/uc-cdis/cdis-manifest/pull/7835/files#diff-aa523d7ebc648ae36c5aa6055cfc73856a89896d2cbfdae4caea0a4b8e372d52R20

**Review by jawadqur — APPROVED** [review]

File: gen3.theanvil.io/portal/gitops.json [hunk 1]
```diff
@@ -1,5 +1,9 @@
 {
   "gaTrackingId": "UA-119127212-17",
+  "grafanaFaroConfig": {
+    "grafanaFaroEnable": true,
+    "grafanaFaroNamespace": "anvilprod"
+  }
```
Comment (jawadqur) [inline gen3.theanvil.io/portal/gitops.json:6]:
```suggestion
  },
```

missing a comma

**Review by jawadqur — CHANGES_REQUESTED** [review]
Missing a comma after faroConfig

**Pull request merged**